### ISSUE TITLE
Does coqbot only look amongst suggested jobs for minimizing?

### ISSUE THREAD

**Issue by JasonGross** [coq/bot#219]
https://github.com/coq/coq/pull/15807#issuecomment-1077917098 :

You requested minimization of suggested failing CI jobs, but no jobs were suggested at commit fb47d4f. You can trigger minimization of ci-analysis with ci minimize all or by requesting some targets by name.

SkySkimmer commented 9 hours ago
@coqbot ci minimize analysis

coqbot-app bot commented 9 hours ago
I was unable to minimize any of the CI targets that you requested.
The requested target 'analysis' could not be found among the jobs .

**Comment by SkySkimmer** [comment]
It may have been having issues with github hooks (this would explain why it didn't respond to my next comment)

**Comment by JasonGross** [comment]
The fact that it commented at all shows that it received the hook.  The sentence "The requested target 'analysis' could not be found among the jobs ." is only emitted when the filtered list of jobs is empty, which makes me think that we should reorganize the logic a bit to explain the issue.

**Comment by SkySkimmer** [comment]
It didn't comment after https://github.com/coq/coq/pull/15807#issuecomment-1077924737

**Comment by JasonGross** [comment]
Right, so I think it was having issues with web hooks, but I also think this is a separate issue unrelated to that issue, because it did respond to your first comment at https://github.com/coq/coq/pull/15807#issuecomment-1077917098
